ReShade/Shaders/Otis/MouseOverlay.fx

```diff
@@ -0,0 +1,33 @@
+///////////////////////////////////////////////////////////////////
+// This effects simply shows the mouse coordinates on the screen. It can be used in combination
+// of effects which use the mousecoordinates supplied by ReShade so you can better see what
+// you're doing.
+///////////////////////////////////////////////////////////////////
+
+#include EFFECT_CONFIG(Otis)
+#include "Common.fx"
+
+#if USE_MOUSEOVERLAY
+
+#pragma message "MouseOverlay by Otis\n"
+
+namespace Otis
+{
+	float4 PS_MouseCoordOverlay(float4 vpos:SV_Position, float2 texcoord: TEXCOORD) :SV_Target
+	{
+		float2 focusPoint = ReShade::MouseCoords * ReShade::PixelSize;
+		if(abs(focusPoint.x - texcoord.x) < MOL_CursorSize && abs(focusPoint.y - texcoord.y) < MOL_CursorSize)
```

Comment:
I can't get it to work with SV_Position, even though it's a DX11 game (AC Syndicate), and the semantics should be 'screencoordinate'. Whatever I do with vpos, it will never result in a pixel being rendered. As a float2 can't be compared with another float2 using `<` it seems, I used:

```c
float2 normalizedMouseCoords = abs(ReShade::MouseCoords - vpos.xy);
return ( normalizedMouseCoords.x < MOL_CursorSize) && (normalizedMouseCoords.y < MOL_CursorSize) ? 
						  float4(MOL_CursorColor, 1.0) 
						: tex2D(ReShade::BackBuffer, texcoord);
```
but that never gets a pixel rendered. (always backbuffer).